cmd/genbindings/transformctors.go

```diff
@@ -4,30 +4,56 @@ import (
 	"sort"
 )
 
-// astTransformConstructorOrder creates a canonical ordering for constructors
-// where the 0th entry is any entry taking solely a QWidget* parameter.
-// This is so that UIC can safely assume this for types.
-// @ref https://github.com/mappu/miqt/issues/46
+// astTransformConstructorOrder creates a canonical ordering for constructors.
 func astTransformConstructorOrder(parsed *CppParsedHeader) {
 
 	// QFoo(QWidget* parent);
-	checkIsDefaultCtor := func(candidate CppMethod) bool {
+	checkIsDefaultCtor := func(candidate *CppMethod) bool {
 		return len(candidate.Parameters) == 1 &&
 			candidate.Parameters[0].ParameterType == "QWidget" &&
 			candidate.Parameters[0].Pointer
 	}
 
+	// QFoo(QVariantList);
+	checkIsQVariantListCtor := func(candidate *CppMethod) bool {
```

Comment:
hairy :grimacing:

Reply:
ie the renumbering same happens for some `connect` overloads, private types, `void*` parameters etc that eventually are needed for example for metaobject support.

Reply:
Yes, of course. This doesn't fix the issue in general, only, mitigates some of the impact of this specific change. If we come up with a general solution it can supersede this one.